uktrade/data-hub-frontend
# Move investment info text to form

## Description of change

The investment type info has been moved into the investment form, and the old investment type info page has been deleted.

## Test instructions

Start the process for adding an investment project. When you get to the investment type section you should see a `Details` section under each type instead of a link to a separate page.

## Screenshots

### Before

Links within the form
<img width="999" alt="Screenshot 2023-08-17 at 13 05 03" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/02898cf3-0e8b-4820-961e-303247601eb2">

The information page itself
![Screenshot 2023-08-17 at 13-05-06 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/7011e9cd-ed44-4baa-8832-fe8fd13f77e5)

### After

![Screenshot 2023-08-17 at 13-04-04 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/abfa784f-f8f8-4d13-bc0b-e3b8866034ae)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
